src/doc/pythonbindings.rst

```diff
@@ -4027,6 +4027,43 @@ following boilerplate:
 
 |
 
+
+Jupyter Notebooks
+-----------------
+
+Like any other Python package, OpenImageIO can be used in `Jupyter notebooks <https://jupyter.org/install>`_. 
```

Comment:
Hi, sorry, one more request for change. 
The "recipes" section is progressing from simple common tasks to more rare or complex examples.
So instead of making this the first item in the section, can you literally make it the very last example (and thus the last text in this file)? Thanks.